lib/token-assert.js

```diff
@@ -38,6 +38,21 @@ TokenAssert.prototype.whitespaceBetween = function(options) {
                 column: token.loc.end.column
             });
         }
+    } else if (options.hasOwnProperty('maximumSpaces')) {
```

Comment:
`atMost` option is quietly fine if method will be about spaces (if we decide to use `atMost` for lines). I just wanted some consistency in options naming.